tests/MassTransit.SignalR.Tests/ScaleoutHubLifetimeManagerTests.cs

```diff
@@ -15,10 +15,13 @@ public class ScaleoutHubLifetimeManagerTests : DoubleScaleoutBackplaneTestFixtur
         async Task AssertMessageAsync(TestClient client)
         {
             var message = await client.ReadAsync().OrTimeout() as InvocationMessage;
-            Assert.NotNull(message);
-            Assert.AreEqual("Hello", message.Target);
-            Assert.AreEqual(1, message.Arguments.Length);
-            Assert.AreEqual("World", message.Arguments[0].ToString());
+            Assert.That(message, Is.Not.Null);
+            Assert.Multiple(() =>
+            {
+                Assert.That(message.Target, Is.EqualTo("Hello"));
+                Assert.That(message.Arguments.Length, Is.EqualTo(1));
+            });
+            Assert.That(message.Arguments[0].ToString(), Is.EqualTo("World"));
```

Comment:
Move in multiple?

Reply:
actually it is correct, we need to fail if there is no items in collection